### ISSUE TITLE
Surface pod condition information in pod lists

### ISSUE THREAD

**Issue by bryk** [kubernetes/dashboard#1353]
The issue https://github.com/kubernetes/dashboard/issues/1317 started a discussion that we should surface pod condition info in pod lists. Or, at leas, pod lists on service pages. 

I.e., this section

```
Conditions:
  Type      Status
  Initialized   True 
  Ready     False 
  PodScheduled  True 
Volumes:
  configuration-volume:
    Type:   ConfigMap (a volume populated by a ConfigMap)
.....
```

I propose that we get rid of Cluster IP column and use this one. Cluster IP for a pod is not worth showing on all lists. This column is useless.

**Comment by IanLewis** [comment]
Do you want to show each condition as a column in the pod list?

This seems similar to the discussion in #1349 to me. I feel like we should be using these conditions to determine the icons we show.

**Comment by bryk** [comment]
> Do you want to show each condition as a column in the pod list?

No. I'd show pod conditions as a single column. The pod can be in only one condition or multiple? 

@rf232

**Comment by IanLewis** [comment]
Not sure why the API returns it as a list rather than an object, but I believe it can only have one condition of each type.

**Comment by rf232** [comment]
I'd most definitely prefer a single column for the status. I don't think we
really have multiple states, and if we have they should be substates and
not orthogonal states afaik.

**Comment by maciaszczykm** [comment]
Do we prefer having all conditions in one column and display only conditions with `True` status or just column per condition?

I like second solution more, because users can see everything, not only things, that are okay. We can use icons instead of `True` and `False`.

**Comment by maciaszczykm** [comment]
> The pod can be in only one condition or multiple?

In multiple:

```
$ kubectl describe pod monitoring-influxdb-grafana-v3-qfyio -n kube-system
Name:		monitoring-influxdb-grafana-v3-qfyio
Namespace:	kube-system
Node:		celcius/192.168.30.254
Start Time:	Thu, 10 Nov 2016 10:17:32 +0100
Labels:		k8s-app=influxGrafana
		kubernetes.io/cluster-service=true
		version=v3
Status:		Running
IP:		172.17.0.3
Controllers:	ReplicationController/monitoring-influxdb-grafana-v3
Containers:
  influxdb:
    Container ID:	docker://ca8cbb2c8ecdb94cf00395ee6a50477b18150ee6ade1882238ab6ef8819b3b43
    Image:		gcr.io/google_containers/heapster_influxdb:v0.7
    Image ID:		docker://sha256:b9a29a864750afa03dc00fc17f3ddaed026793fec104af2ad12b9fb4e2f889f1
    Ports:		8083/TCP, 8086/TCP
    Limits:
      cpu:	100m
      memory:	500Mi
    Requests:
      cpu:		100m
      memory:		500Mi
    State:		Running
      Started:		Thu, 10 Nov 2016 10:17:50 +0100
    Ready:		True
    Restart Count:	0
    Volume Mounts:
      /data from influxdb-persistent-storage (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-2t0bd (ro)
    Environment Variables:	<none>
  grafana:
    Container ID:	docker://9b7331503a3f026a4795864eba81a22e5e9bcd7efd396357a64bec2b38348923
    Image:		gcr.io/google_containers/heapster_grafana:v3.1.1
    Image ID:		docker://sha256:41b92a01197f7605d5805f8a7bbebf8f4972c635662e701dbedb03bc58c71412
    Port:		
    Limits:
      cpu:	100m
      memory:	100Mi
    Requests:
      cpu:		100m
      memory:		100Mi
    State:		Running
      Started:		Thu, 10 Nov 2016 10:18:11 +0100
    Ready:		True
    Restart Count:	0
    Volume Mounts:
      /var from grafana-persistent-storage (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-2t0bd (ro)
    Environment Variables:
      INFLUXDB_SERVICE_URL:		http://monitoring-influxdb:8086
      GF_AUTH_BASIC_ENABLED:		false
      GF_AUTH_ANONYMOUS_ENABLED:	true
      GF_AUTH_ANONYMOUS_ORG_ROLE:	Admin
      GF_SERVER_ROOT_URL:		/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana/
Conditions:
  Type		Status
  Initialized 	True 
  Ready 	True 
  PodScheduled 	True 
Volumes:
  influxdb-persistent-storage:
    Type:	EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:	
  grafana-persistent-storage:
    Type:	EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:	
  default-token-2t0bd:
    Type:	Secret (a volume populated by a Secret)
    SecretName:	default-token-2t0bd
QoS Class:	Guaranteed
Tolerations:	<none>
No events.
```

**Comment by maciaszczykm** [comment]
Closed #1444 as we agreed for a different solution discussed there.

**tl;dr** Conditions should be surfaced as an status icon in list views.
